components/operator/internal/modules/pod.go

```diff
@@ -39,7 +39,7 @@ func (d *defaultPodDeployer) deploy(ctx context.Context, pod pod) error {
 			matchLabels := collectionutils.CreateMap("app.kubernetes.io/name", pod.name)
 			strategy := appsv1.DeploymentStrategy{}
 			if pod.disableRollingUpdate {
```

Comment:
Strange @flemzord ^^
the configuration say "disableRollingUpdate", and you set the strategy "RollingUpdateDeploymentStrategyType".